doc/man3/OCSP_sendreq_new.pod

```diff
@@ -134,12 +129,14 @@ The OCSP-specific names for types and functions were deprecated in OpenSSL 3.0.
 
 The B<OCSP_REQ_CTX> typedef is an alias for B<OSSL_HTTP_REQ_CTX>.
 
-The OCSP_REQ_CTX_set1_req(), OCSP_REQ_CTX_free(),
+The
+OCSP_REQ_CTX_i2d(),
 OCSP_REQ_CTX_add1_header(),
-OCSP_REQ_CTX_set1_req(),
-OCSP_set_max_response_length()
-functions were deprecated in OpenSSL 3.0. OCSP_REQ_CTX_set1_req(rctx, req)
-is equivalent to the following:
+OCSP_REQ_CTX_free(),
+OCSP_set_max_response_length(),
+and OCSP_REQ_CTX_set1_req(),
```

Comment:
Please make also their order consistent with the above.
Nit: remove trailing `,`

Reply:
The order does match the NAME and SYNOPSIS sections so I don't know what you want.  Removed the trailing comma.

Reply:
Ah, right, the order is already consistent. Maybe Santa confused me :wink:

Yet i just noticed one last thing:
```
OCSP_REQ_CTX_set1_req(rctx, req) is equivalent to the following:

 OSSL_HTTP_REQ_CTX_i2d(rctx, "application/ocsp-request",
                       ASN1_ITEM_rptr(OCSP_REQUEST), (ASN1_VALUE *)req)
```
would fit better at the end of the DESCRIPTION section, 
mainly because otherwise `OCSP_REQ_CTX_set1_req()` is not mentioned there at all.

Once this is done, I think you can squash all commits since at least in my view it will be finished.

Reply:
> Once this is done, I think you can squash all commits since at least in my view it will be finished.

Moved the text and squashed.  Thanks for the review!